src/net40/Radical.Windows.Presentation/Regions/Specialized/TabControlRegion.cs

```diff
@@ -39,16 +39,27 @@ public TabControlRegion( string name )
         /// <param name="view">The view.</param>
         protected override void OnAdd(DependencyObject view)
         {
-            var tabItem = new TabItem();
-            tabItem.Content = view;
+            if (view == null) throw new ArgumentNullException(nameof(view));
```

Comment:
Drop this check, it should be added here https://github.com/RadicalFx/Radical.Windows.Presentation/blob/develop/src/net40/Radical.Windows.Presentation/Regions/ElementsRegion.cs#L62

Reply:
The check and also the creation of the tab item that then we pass to this code? 

'			this.OnAdd( view );
			this.views.Add( view );
			this.OnAdded( view );
'

Reply:
The check only, but should not be in this PR. Just remove it.